tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -130,3 +131,29 @@ Verify that the amended methodology content is correct
     ${content}=    user gets accordion section content element    New and Updated Title
     user checks element contains    ${content}    Adding Methodology content
     user checks element contains    ${content}    New & Updated content
+
+Schedule a methodology amendment to be published with a release amendment
+    user navigates to admin dashboard
+    user creates amendment for release    ${PUBLICATION_NAME}    ${RELEASE_NAME}    (Live - Latest release)
+    user creates methodology amendment for publication    ${PUBLICATION_NAME}
+    user approves methodology amendment for publication
+    ...    publication=${PUBLICATION_NAME}
+    ...    publishing_strategy=WithRelease
+    ...    with_release=${PUBLICATION_NAME} - ${RELEASE_NAME}
+    ${details}=    user opens release summary on the admin dashboard    ${PUBLICATION_NAME}    ${RELEASE_NAME}
```

Comment:
Feels like there could be a break here and split this up into two tests. Something like `Verify cancelling the dependant release leaves the Methodology as Draft`